Header header
string qr_message
geometry_msgs/Point[] bounding_polygon
